_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-20 12:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+author: Kyle Begovich
```

Comment:
Let's add a cover image. If you have one in mind we can use it, or we can see if we can get something from Design.

Reply:
have done this, just a cropped picture of the landing page